lib/matplotlib/tests/test_style.py

```diff
@@ -135,6 +152,23 @@ def test_context_with_badparam():
         assert mpl.rcParams[PARAM] == other_value
 
 
+def test_parent_stylesheets():
+    style_dir = os.path.join(os.path.dirname(__file__), 'styles')
+    with temp_style_dir(style_dir):
+        with style.context('main_test'):
+            assert mpl.rcParams['lines.linewidth'] == 5.0
+            assert mpl.rcParams['lines.marker'] == 'o'
+            assert mpl.rcParams['lines.linestyle'] == '--'
+            assert mpl.rcParams['lines.markersize'] == 12
+            assert mpl.rcParams['axes.facecolor'] == 'green'
+            assert mpl.rcParams['axes.linewidth'] == 5.0
+            assert mpl.rcParams['axes.grid'] == True
+            assert mpl.rcParams['axes.labelweight'] == 'bold'
+            assert mpl.rcParams['axes.labelcolor'] == 'yellow'
+            assert mpl.rcParams['axes.color_cycle'] == [u'r', u'g', u'b',
+                                                            u'c', u'y']
```

Comment:
As mentioned in my overall PR comment, it's difficult to actually verify this test because the input data is in a separate file (i.e. disconnected from the test code). Also, it's probably not necessary to test all of these values. For most tests, I think checking one value is sufficient.